VultisigApp/VultisigApp/Extensions/StringExtension.swift

```diff
@@ -135,3 +135,42 @@ extension String {
         return formattedString
     }
 }
+
+extension String {
+    init(hex: String) {
+        let cleanedHex = hex.stripHexPrefix()
+        var str = ""
+        var index = cleanedHex.startIndex
+        while index < cleanedHex.endIndex {
+            let nextIndex = cleanedHex.index(index, offsetBy: 2)
+            if nextIndex <= cleanedHex.endIndex {
+                let byte = cleanedHex[index..<nextIndex]
+                if let char = UInt8(byte, radix: 16).map({ Character(UnicodeScalar($0)) }) {
+                    str.append(char)
+                }
+                index = nextIndex
+            } else {
+                break
+            }
+        }
+        self = str
+    }
+    func hexToData() -> Data? {
```

Comment:
you can just do `Data(hexString:xxxx)`  to decode hexstring to data

Reply:
Please remove this `StringExtensions.swift` , and use existing methods

Reply:
Ok, so the idea is converting from hex to data then data to utf